src/main.js

```diff
@@ -1 +1,22 @@
-'use strict';
+// eslint-disable-next-line strict
+function updateClock() {
+  const now = new Date();
+
+  const hourHand = document.querySelector('.hour-hand');
+  const minuteHand = document.querySelector('.minute-hand');
+  const secondHand = document.querySelector('.second-hand');
+
+  const hours = now.getHours();
+  const minutes = now.getMinutes();
+  const seconds = now.getSeconds();
+
+  const hourRotation = (hours % 12) * 30 + minutes / 2;
+  const minuteRotation = minutes * 6;
+  const secondRotation = seconds * 6;
+
+  hourHand.style.transform = `rotate(${hourRotation}deg)`;
+  minuteHand.style.transform = `rotate(${minuteRotation}deg)`;
+  secondHand.style.transform = `rotate(${secondRotation}deg)`;
+}
+
+setInterval(updateClock, 1000);
```

Comment:
Since the task requires to create a stopwatch with only CSS animations, the setInterval call to update the clock with JavaScript should be removed.